src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,21 @@
-export const TodoInfo = () => {};
+import { UserInfo } from '../UserInfo/UserInfo';
+import { TodoInfoType } from '../../types/TodoInfoType';
+import React from 'react';
+
+type Props = {
+  todo: TodoInfoType;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={
+        todo.completed === true ? 'TodoInfo TodoInfo--completed' : 'TodoInfo'
```

Comment:
```suggestion
        todo.completed === true ? 'TodoInfo TodoInfo--completed' : 'TodoInfo'
```
- `todo.completed` is already boolean, you don't need to compare with `true`
- To set classes conditionally use [classnames](https://www.npmjs.com/package/classnames) or [clsx](https://www.npmjs.com/package/clsx) library